OpenRA.Mods.Common/Traits/World/ScriptLobbyDropdown.cs

```diff
@@ -23,9 +23,11 @@ public class ScriptLobbyDropdownInfo : TraitInfo, ILobbyOptions
 		public readonly string ID = null;
 
 		[FieldLoader.Require]
+		[TranslationReference]
 		[Desc("Descriptive label for this option.")]
 		public readonly string Label = null;
 
+		[TranslationReference]
```

Comment:
Take a look at how `ActorReferenceAttribute` implements `DictionaryReference`, add the same to `TranslationReferenceAttribute`, then add `[TranslationReference(dictionaryReference: LintDictionaryReference.Values)]` to `Values` below.

This should fix the missing linting.